packages/zk-eddsa-event-ticket-pcd/test/ZKEdDSAEventTicketPCD.spec.ts

```diff
@@ -303,6 +324,37 @@ describe("ZKEdDSAEventTicketPCD should work", function () {
     expect(verificationRes).to.be.true;
   });
 
+  it("should reveal semaphore ID, ticketCategory, timestamps, and reserved fields if requested, and no more", async function () {
+    const pcdArgs = await toArgs(ticketData1, fieldsToReveal3, true);
+    const pcd = await ZKEdDSAEventTicketPCDPackage.prove(pcdArgs);
+
+    const claim = pcd.claim;
+    expect(claim.partialTicket.attendeeSemaphoreId).to.be.equal(
```

Comment:
Might be cleaner to construct your own PartialTicket and do a deepequals comparison?  That way you can't miss any fields by accident?